console/src/components/Services/Data/TableInsertItem/InsertActions.js

```diff
@@ -23,8 +27,40 @@ const I_FETCH_ENUM_OPTIONS_ERROR = 'InsertItem/I_FETCH_ENUM_ERROR';
 const Open = () => ({ type: _OPEN });
 const Close = () => ({ type: _CLOSE });
 
+// insert helper
+const createInsertMigration = (dispatch, getState, tableInfo, insertObj) => {
+  const upQuery = getInsertQuery(
+    { name: tableInfo.name, schema: tableInfo.schema },
+    insertObj
+  );
+  const migrationName = `insert_into_${tableInfo.name}`;
+  const customOnSuccess = () => {
+    // NOTE: maybe unnecessary.
+    dispatch(setTable(tableInfo.name));
+  };
+  const customOnError = () => {};
+  const requestMessage = 'Creating Insert Migration';
+  const successMessage = 'Created Insert Migration!';
+  const errorMessage = 'Error in creating Insert Migration';
+
+  makeMigrationCall(
+    dispatch,
+    getState,
+    [upQuery],
+    null,
+    migrationName,
+    customOnSuccess,
+    customOnError,
+    requestMessage,
+    successMessage,
+    errorMessage,
+    true,
```

Comment:
unsure
should this be `true`? - this `true` value corresponds to `shouldSkipSchemaReload`.

Reply:
Need more context on this @beerose